config/routes.rb

```diff
@@ -43,7 +41,7 @@
   resources :holds do
     post :lift, on: :member
   end
-  resources :financial_transaction, except: %i(destroy update)
```

Comment:
I'm basing changes on this line as opposed to the above line because Metin said he did not want Financial Transactions to be deleted or edited after creation